src/python/grpcio/grpc/_channel.py

```diff
@@ -1322,15 +1465,16 @@ def __init__(self, channel):
         self.callbacks_and_connectivities = []
         self.delivering = False
 
-    def reset_postfork_child(self):
+    def reset_postfork_child(self) -> None:
         self.polling = False
         self.connectivity = None
         self.try_to_connect = False
         self.callbacks_and_connectivities = []
         self.delivering = False
 
 
-def _deliveries(state):
+def _deliveries(
```

Comment:
Why `Optional`? Should this be `List[Callable[[grpc.ChannelConnectivity], None]]`?

Reply:
Yes, fixed it.